A32NX/ModelBehaviorDefs/A32NX/Interior/A32NX_Interior_Misc.xml

```diff
@@ -249,6 +249,79 @@
         </UseTemplate>
```

Comment:
You can probably just revert this file to what it is in master, no ? I think you told me about this being broken

Reply:
reverted to master